lib/model/model.go

```diff
@@ -1912,12 +1939,18 @@ func sendIndexes(conn protocol.Connection, folder string, fs *db.FileSet, prevSe
 		// currently in the database, wait for the local index to update. The
 		// local index may update for other folders than the one we are
 		// sending for.
-		if fs.Sequence(protocol.LocalDeviceID) <= prevSequence {
-			sub.Poll(time.Minute)
+		if s.fset.Sequence(protocol.LocalDeviceID) <= s.prevSequence {
+			select {
+			case <-s.stop:
+				return
+			case <-evChan:
```

Comment:
This'll burn a bit more CPU as you'll now loop and create a one minute timer on every event, eventually having potentially hundreds of thousands of timers running in the background. I don't think we need the timer any more either given there's the select on the `stop` in parallel. If we do, a `time.Ticker` declared once outside the loop should do.